jans-cedarling/cedarling/src/authz/entities/entity_create.rs

```diff
@@ -0,0 +1,140 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use std::{
+    collections::{HashMap, HashSet},
+    str::FromStr,
+};
+
+use crate::models::token_data::{GetTokenClaimValue, TokenClaim};
+use cedar_policy::{EntityId, EntityTypeName, EntityUid, RestrictedExpression};
+
+/// Meta information about an entity type.
+/// Is used to store in `static` variable.
+pub struct EntityMetaInfo<'a> {
```

Comment:
just a suggestion, not necessarily a correction.
what do you think about renaming this to `EntityMetadata`

Reply:
thanks, updated